Makefile

```diff
@@ -123,18 +129,19 @@ endif
 
 # Explicit clean
 .PHONY: clean
-clean:
-	@if [ -d deps/jerryscript ]; then \
-		make -C $(JERRY_BASE) -f targets/zephyr/Makefile clean BOARD=$(BOARD); \
-		rm -rf deps/jerryscript/build/$(BOARD)/; \
-		rm -rf deps/jerryscript/build/lib; \
-	fi
-	@if [ "$(TARGET)" != "linux" ]; then \
+clean: update
```

Comment:
Since I found at least one other thing to comment about, how about add a comment here as to why you're doing this - so we're more likely to be able to clean it up later. Namely, this update makes clean work on a brand new repo.